README.md

```diff
@@ -1 +1,49 @@
 # blocks-concurrent-batch-subscriber
+
+## Overview
+
+`blocks-concurrent-subscriber` subscribes the progresses of jobs with `blocks concurrent batch board` of [magellan-blocks](https://www.magellanic-clouds.com/blocks/).
+When the progresses are notified, `blocks-concurrent-subscriber` updates the status of the job and inserts the logs on mysql.
+
+`blocks-concurrent-subscriber` access to `blocks-concurrent-batch-agent` to get subscriptions to pull messages.
+
+## Install
+
+```bash
+$ go get github.com/groovenauts/blocks-concurrent-subscriber
+```
```

Comment:
This installation method ignores `glide.lock`.
I think it would be better to include `vendor` directory or use `glide install && go build`, and prepare precompiled binaries at github releases, _in the future_.